haskell.el

```diff
@@ -398,18 +398,57 @@ Give optional NEXT-P parameter to override value of
          (buffer (haskell-session-interactive-buffer session)))
     (pop-to-buffer buffer)))
 
+
+(defun haskell--file-name-to-load-string (file-name)
+  "Create a GHCi repl load statement from FILE-NAME."
+  (format "load \"%s\"" (replace-regexp-in-string
+                         "\""
+                         "\\\\\""
+                         file-name)))
+
+
+(defvar haskell-process-do-hsc2hs t
```

Comment:
We would not want to push the burden on users with this variable due to cost described in https://github.com/haskell/haskell-mode/wiki/Customizable-variables